Snippets/Metrics/Metrics_1/Metrics_1.csproj

```diff
@@ -4,7 +4,6 @@
     <TargetFramework>net462</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="6.2.1" />
-    <PackageReference Include="NServiceBus.Metrics" Version="1.0.0" />
+    <PackageReference Include="NServiceBus.Metrics" Version="1.1.0" />
```

Comment:
This is a minor so it's ok to have it 1.1 as well (snippets versioned by project name/dir structure). Should I change it to 1.0?